### DISCUSSION TITLE
Show chapters for streams only

### DISCUSSION THREAD

**Discussion by Solarunit** [tomasklaen/uosc#867] [Q&A] [answered]
Is it possible to show chapters for streams (youtube videos) only and do not show for other videos?

**Comment by tomasklaen** [comment]
No, too niche.

**Reply by Solarunit** [reply]
I see, I thought there was some option depending on type of video out there

**Comment by FichteFoll** [comment, marked as answer]
I suppose you could make the chapters invisible by setting their opacity to `0.0` via the `opacity` setting and use [auto-profiles](https://mpv.io/manual/stable/#conditional-auto-profiles) for this?

Something like (untested)

```
[stream]
profile-cond= get("path", ""):find("youtube%.com/") ~= nil
profile-restore=copy-equal
script-opts-append=uosc-opacity=chapters=0
```

**Reply by christoph-heinrich** [reply]
`profile-cond=demuxer_via_network` also works

**Reply by Solarunit** [reply]
interesting, I will try it, thanks!

**Reply by Solarunit** [reply]
So in mpv.conf I added it in section:

```
[cache]
profile-cond=demuxer_via_network
script-opts-append=uosc-opacity=chapters=0.5
...
```
in [default] section above I have this:
`script-opts-append=uosc-opacity=chapters=0`

and looks like it works, thanks!
